index.html

```diff
@@ -1,111 +1,181 @@
-
-              <div class="name">
-                <p>AYAKO YOKOYA</p> <!-- Name -->
-              </div>
-              <div class="aboutme">
-                <p>Creative Developer <span>(I wanna be...)</span></p> <!-- Profession -->
-              </div>
-
-              <div class="sns-btn-box">
-                <p>My sns accounts are...</p>
-                <a
-                  href="https://github.com/ayakoyokoya/"
-                  target="_blank"
-                  class="sns-btn"
-                >
-                  github <!-- Link to social media 1 -->
-                </a>
-                <a
-                  href="https://twitter.com/ayakoyokoya"
-                  target="_blank"
-                  class="sns-btn"
-                >
-                  twitter <!-- Link to social media 2 -->
-                </a>
-              </div>
-
-              <div class="portfolio-btn-box">
-                <p>
-                  Do you want to see error? <!-- Any text -->
-                </p>
-                <div class="portfolio">
-                  <button id="error-btn" class="error-btn">
-                    error?
-                  </button>
-                  <a
-                    href="https://ayakoyokoya.github.io/yaba-kawaii-audio-visualizer/dist/"
-                    target="_blank"
-                    class="portfolio-btn"
-                  >
-                    yes! 
-                  </a>
-                </div>
-              </div>
-            </div>
-          </div>
+<body class="body">
+  <header class="header">
+    <img src="img/Logos.svg" alt="" class="logo">
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li class="navigation-item"><a href="" class="nav-link">Home</a></li>
+        <li class="navigation-item"><a href="blog-article.html" class="nav-link">Blog</a></li>
+        <li class="navigation-item"><a href="" class="nav-link">Catalog</a></li>
+        <li class="navigation-item"><a href="" class="nav-link">Contact Us</a></li>
+      </ul>
+    </nav>
+    <button class="search-bar" type="button">Search</button>
+    <a href="https://discord.com" target="_blank" class="discord">Discord</a>
+    <button class="connect" type="button">Connect</button>
+  </header>
+  <main class="content">
+    <section class="headline">
+      <h1 class="headline-text">With <strong>Binabox</strong> Explore NFT collection</h1>
+      <p class="title-description">We are the best way to check the rarity of NFT collection</p>
+      <ul class="achievements">
+       <li class="achievement-info">2240+ Total item</li>
+       <li class="achievement-info">1000+ Profile whitelisted</li>
+      </ul>
+    </section>
+    <section class="catalog">
+      <h2 class="section-name">Catalog</h2>
+      <a href="" class="see-all">See all</a>
+      <button class="move-right" type="button">Right</button>
+      <button class="move-left" type="button">Left</button>
+      <article class="catalog-items">
```

Comment:
@Alinchik1 this is the catalog, it's the list of element(products), and each element represents a self-contained composition. So I think the structure we need, like:
`<ul class="catalog-list">
    <li class="catalog-item">
        <article class="catalog-product">...</article>
    </li>
    <li class="catalog-item">
        <article class="catalog-product">...</article>
    </li>
</ul>
`

Reply:
done